src/main/java/Duke.java

```diff
@@ -1,10 +1,46 @@
+import duke.task.TaskException;
+import duke.task.TaskList;
+import duke.util.FileManager;
+import duke.util.Parser;
+import duke.util.Ui;
+
+import java.util.*;
```

Comment:
I think you should avoid wildcard imports and instead list out all the imports that you require.